runners/apex/pom.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.beam</groupId>
+    <artifactId>beam-runners-parent</artifactId>
+    <version>0.3.0-incubating-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>beam-runners-apex_3.4.0</artifactId>
+
+  <name>Apache Beam :: Runners :: Apex</name>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <apex.core.version>3.4.0</apex.core.version>
+    <apex.malhar.version>3.4.0</apex.malhar.version>
+    <skipIntegrationTests>true</skipIntegrationTests>
```

Comment:
The tests definitely don't all need to pass. As long as the right set is selected by the configuration here, so we know what works and what doesn't, the PR can merge to a feature branch.

Even for our runners in `master` they don't all pass. We just relate the tests that pass to the capability matrix.

So, yes, please do blacklist tests that will take a long time to pass. It is better to blacklist them by JUnit categories instead of by test name. You may need to create some new categories to describe the features being blacklisted.